applications/asset_tracker_v2/src/modules/modem_module.c

```diff
@@ -80,6 +81,16 @@ const k_tid_t module_thread;
 K_MSGQ_DEFINE(msgq_modem, sizeof(struct modem_msg_data),
 	      MODEM_QUEUE_ENTRY_COUNT, MODEM_QUEUE_BYTE_ALIGNMENT);
 
+#if defined(CONFIG_NRF_MODEM_LIB)
+K_SEM_DEFINE(nrf_modem_initialized, 0, 1);
```

Comment:
At some point `nrf_modem_lib_init` should probably be called from the `modem_module` instead of `main.c`